src/NuGet.Services.SearchService/App_Start/WebApiConfig.cs

```diff
@@ -40,11 +47,31 @@ public static void Register(HttpConfiguration config)
                 routeTemplate: "search/query",
                 defaults: new
                 {
-                    controller = "Search",
+                    controller = GetControllerName<SearchController>(),
                     action = nameof(SearchController.V2SearchAsync),
                 });
 
+            config.Routes.MapHttpRoute(
+                name: "V3Search",
+                routeTemplate: "query",
+                defaults: new
+                {
+                    controller = GetControllerName<SearchController>(),
+                    action = nameof(SearchController.V3SearchAsync),
+                });
+
             config.EnsureInitialized();
+
+            HostingEnvironment.QueueBackgroundWorkItem(token => ReloadAuxiliaryFilesAsync(dependencyResolver, token));
+        }
+
+        private static async Task ReloadAuxiliaryFilesAsync(IDependencyResolver dependencyResolver, CancellationToken token)
+        {
+            using (var scope = dependencyResolver.BeginScope())
+            {
+                var loader = (IAuxiliaryFileReloader)scope.GetService(typeof(IAuxiliaryFileReloader));
+                await loader.ReloadContinuouslyAsync(token);
```

Comment:
We've had issues where a database context was used in a scope that was never recycled. I know this doesn't use a database context today, but I'm a little concerned about this scope's lifetime. Do we even need a scope here? It seems the loader doesn't depend on any scoped services.

Reply:
Sure, I think I was hoping to isolate the background worker as much as possible from the root scope, but it's probably better to keep this simple until we have a problem.